uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -744,10 +775,10 @@ sub dicom_to_minc {
     ############################################################
     #### use some other converter if specified in the config ###
     ############################################################
-    if ($converter ne 'dcm2mnc') {
+    if ($converter !~ /dcm2mnc/) {
```

Comment:
This change allows project to modify their prod files to use a different version of dcm2mnc and still use the same command format.  For example dcm2mnc4ng does match the proper command string now.